bin/create_exercise.dart

```diff
@@ -6,87 +6,95 @@ import 'package:args/args.dart';
 import 'package:path/path.dart' show dirname;
 
 /// Constants
```

Comment:
Suggestion: Use `//` when using comments as code separators, otherwise `scriptFilename` gets the doc comment as `Constants`

Reply:
Okay, good point. I don't think I added it in this PR, but I can make the change.

Reply:
I thought since we are already working on it, why not make such adjustments as well. We usually don't create PRs for such refactorings and as a result they remain in the code.